src/sagemaker/instance_types_gpu_info_config/us-west-2.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Are these configs generated or created by you? 

Can they be combined into a single config? 

Let's check with Kalyani if this works in proposed RBA pipeline

Reply:
Created by me, and they can be combined into a single config.

Reply:
I think it needs to be a single config, let's confirm

Reply:
Will combine all into something like below
`
{
  "af-south-1": {
        "ml.p5.48xlarge": {"Count": 8, "TotalGpuMemoryInMiB": 655360},
        "ml.p4d.24xlarge": {"Count": 8, "TotalGpuMemoryInMiB": 327680},
        "ml.p4de.24xlarge": {"Count": 8, "TotalGpuMemoryInMiB": 655360},
        "ml.p3.2xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 16384},
        "ml.p3.8xlarge": {"Count": 4, "TotalGpuMemoryInMiB": 65536},
        "ml.p3.16xlarge": {"Count": 8, "TotalGpuMemoryInMiB": 131072},
        "ml.p3dn.24xlarge": {"Count": 8, "TotalGpuMemoryInMiB": 262144},
        "ml.p2.xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 12288},
        "ml.p2.8xlarge": {"Count": 8, "TotalGpuMemoryInMiB": 98304},
        "ml.p2.16xlarge": {"Count": 16, "TotalGpuMemoryInMiB": 196608},
        "ml.g4dn.xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 16384},
        "ml.g4dn.2xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 16384},
        "ml.g4dn.4xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 16384},
        "ml.g4dn.8xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 16384},
        "ml.g4dn.16xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 16384},
        "ml.g4dn.12xlarge": {"Count": 4, "TotalGpuMemoryInMiB": 65536},
        "ml.g5n.xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 24576},
        "ml.g5.2xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 24576},
        "ml.g5.4xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 24576},
        "ml.g5.8xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 24576},
        "ml.g5.16xlarge": {"Count": 1, "TotalGpuMemoryInMiB": 24576},
        "ml.g5.12xlarge": {"Count": 4, "TotalGpuMemoryInMiB": 98304},
        "ml.g5.24xlarge": {"Count": 4, "TotalGpuMemoryInMiB": 98304},
        "ml.g5.48xlarge": {"Count": 8, "TotalGpuMemoryInMiB": 196608}
  },
  ...
}
`

Reply:
We would require a single JSON file. Right now the RBA takes just the `src/sagemaker/image_uri_config` folder for updates. not any other folder.